examples/BasicExample/package-lock.json

```diff
@@ -0,0 +1,5365 @@
+{
```

Comment:
Please remove this package-lock.json . As yarn and yarn.lock is used by default by react-native.

Reply:
Thanks @marcosprosperiOrigin . Please look through the above comment.